### ISSUE TITLE
fedora silverblue python-validity doesn't start

### ISSUE THREAD

**Issue by codebam** [uunicorn/python-validity#95]
Doesn't start on silverblue due to trying to write to `/usr` which is read only on silverblue

**Comment by uunicorn** [comment]
Yes, there's an open PR to fix it, but I think some work is still required. Feel free to contribute:
https://github.com/uunicorn/python-validity/pull/92#issuecomment-890463731
